Create ui-c8y-1023-78-4-add-compact-mode-for-asset-table-v-2.md (#4665)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-78-4-add-compact-mode-for-asset-table-v-2.md

---
date: ""

> title: Add compact mode option for the new asset table v2

Reviewer: ```suggestion
title: Added compact mode option for the new Asset table v2
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66755
version: 1023.78.4
---

> The asset table v2 now includes a compact mode option that will hide the grid header and footer. With this update, you can toggle to compact mode to see more assets on smaller sized widgets.

Reviewer: ```suggestion
The Asset table v2 feature now includes a compact mode option that will hide the grid header and footer. With this update, you can toggle to compact mode to see more assets on smaller-sized widgets.
```

> The Asset table v2 feature now includes a compact mode option that will hide the grid header and footer. With this update, you can toggle to compact mode to see more assets on smaller-sized widgets.

Reviewer: ```suggestion
The Asset table v2 feature now includes a compact mode option that hides the grid header and footer. With this change, you can now toggle to compact mode to see more assets on smaller-sized widgets.
```
